cfme/tests/configure/test_access_control.py

```diff
@@ -3024,4 +3051,35 @@ def test_ansible_playbook_stdout():
             9.
             10. able to see standard output
     """
-    pass
+    user, ansible_catalog = user_catalog
+    service_catalog = ServiceCatalogs(appliance, ansible_catalog, ansible_catalog_item.name)
+    ansible_service = MyService(appliance, ansible_catalog_item.name)
+
+    with user:
+        with appliance.context.use(ViaSSUI):
+            appliance.server.login(user)
+            view = ssui_nav(service_catalog, "Details")
+            view.add_to_shopping_cart.click()
+            service_catalog.order()
+
+    @request.addfinalizer
+    def _finalize():
+        if ansible_service_request.exists():
+            ansible_service_request.remove_request()
+        if ansible_service.exists:
+            ansible_service.delete()
+        ansible_catalog.delete_if_exists()
+
+    with user:
```

Comment:
Have you tried putting everything under `with user` block?
```
with user:
    with appliance.context.use(ViaSSUI):
            appliance.server.login(user)
            ...
            ...
    error_message = (
            "MIQ(Api::TasksController.api_error) Api::NotFoundError: Couldn't find MiqTask with"
        )
        with LogValidator(
                "/var/www/miq/vmdb/log/api.log", failure_patterns=[error_message],
        ).waiting(timeout=120):
            ...
            ...
```

Reply:
No because context is different.